### ISSUE TITLE
[BUG] Cannot install on Mac OS X High Sierra

### ISSUE THREAD

**Issue by davideanastasia** [koaning/scikit-lego#376]
I am currently using a pipenv environment with Python 3.7.7 originally sourced via Homebrew.
When I run `pipenv install scikit-lego`, I get an error which renders the installation impossibile:

```[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/cli/command.py", line 254, in install
[pipenv.exceptions.InstallError]:       editable_packages=state.installstate.editables,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 1992, in do_install
[pipenv.exceptions.InstallError]:       skip_lock=skip_lock,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 862, in do_install_dependencies
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, False, failed_deps_queue, retry=False)
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: []
[pipenv.exceptions.InstallError]: ['ERROR: Could not find a version that satisfies the requirement numba==0.50.1 (from -r /var/folders/sw/q7t3n56s1pq2s460b70k962m0000gn/T/pipenv-p95m9hlh-requirements/pipenv-418l5mdi-requirement.txt (line 1)) (from versions: 0.1, 0.2, 0.3, 0.5.0, 0.6.0, 0.7.0, 0.7.1, 0.7.2, 0.8.0, 0.8.1, 0.9.0, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.12.1, 0.12.2, 0.13.0, 0.13.2, 0.13.3, 0.13.4, 0.14.0, 0.15.1, 0.16.0, 0.17.0, 0.18.1, 0.18.2, 0.19.1, 0.19.2, 0.20.0, 0.21.0, 0.22.0, 0.22.1, 0.23.0, 0.23.1, 0.24.0, 0.25.0, 0.26.0, 0.27.0, 0.28.1, 0.29.0, 0.30.0, 0.30.1, 0.31.0, 0.32.0, 0.33.0, 0.34.0, 0.35.0, 0.36.1, 0.36.2, 0.37.0, 0.38.0, 0.38.1, 0.39.0, 0.40.0, 0.40.1, 0.41.0, 0.42.0, 0.42.1, 0.43.0, 0.43.1, 0.44.0, 0.44.1, 0.45.0, 0.45.1, 0.46.0, 0.47.0, 0.48.0, 0.49.0, 0.49.1rc1, 0.49.1, 0.50.0rc1, 0.50.0)', 'ERROR: No matching distribution found for numba==0.50.1 (from -r /var/folders/sw/q7t3n56s1pq2s460b70k962m0000gn/T/pipenv-p95m9hlh-requirements/pipenv-418l5mdi-requirement.txt (line 1))']```

**Comment by koaning** [comment]
I've never used `pipenv` before, just to check ... if you install via pip/conda everything runs fine?

**Comment by koaning** [comment]
What occurs to me as strange is that our project does not depend on `numba` so it is bound to be one of our dependencies that requires this numba version. Possibly it is `umap-learn` but I can't be sure.

**Comment by davideanastasia** [comment]
I works fine with conda on my other Mac with Catalina (just tested)
Installing 0.4.3 on my Mac with High Sierra in a pipenv environment works fine too

**Comment by MBrouns** [comment]
umap-learn does require numba, but it's quite a lenient dependency (`"numba >= 0.46, != 0.47"`). Also, numba 0.50.1 is released on PyPI, so it's strange that it couldn't be found. AFAIK, pipenv uses PyPI

**Comment by koaning** [comment]
You could try manually installing numba 0.46 and then installing scikit-lego. That should work.

**Comment by davideanastasia** [comment]
Thanks for your help: I managed to successfully install the latest version of `scikit-lego` initially forcing numba to 0.46 and tbb to 2019.0. After playing a little bit, I managed to install numba 0.50.0 (which is just one behind the latest 0.50.1), so I am wondering if there is a bit of delay in getting modules ready for Mac on PyPI?

Anyway, I think it's fair to say this is not a bug in scikit-lego, so I am closing.
